custom/enikshay/case_utils.py

```diff
@@ -169,7 +172,7 @@ def get_all_episode_cases_from_person(domain, person_case_id):
     return episode_cases
 
 
-def get_open_episode_case_from_occurrence(domain, occurrence_case_id):
+def get_open_active_episode_case_from_occurrence(domain, occurrence_case_id):
     """
     Gets the first open 'episode' case for the occurrence
```

Comment:
Lets update the comment to indicate that this is for public sector DSTB episodes